addons/purchase_stock/tests/test_product_template.py

```diff
@@ -0,0 +1,30 @@
+from odoo.tests.common import TransactionCase
+
+
+class TestProductTemplate(TransactionCase):
+  def setUp(self):
+      super(TestProductTemplate, self).setUp()
+
+      self.partner = self.env['res.partner'].create({
+        'name': 'Azure Interior',
+      })
+
+      self.seller = self.env['product.supplierinfo'].create({
+        'name': self.partner.id,
+        'price': 12.0,
+        'delay': 1,
+        'product_code': 'VOB2a',
+      })
+
+      self.product_tmpl = self.env['product.template'].create({
+        'name': 'Rubber Duck',
```

Comment:
I think you made a typo:
```suggestion
        'name': 'Rubber Dick',
```